### PR TITLE
small dashboard refresh

### PR DESCRIPTION
**Pull request by orthagh**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes

A quick refresh of the dashboard cards:
- make round borders visible
- add a subtle border color for ligh big numbers
- remove shadow on cards

image

### CODE REVIEW

File: src/Toolbox.php [hunk 1]
```diff
@@ -3006,6 +3006,35 @@ public static function getFgColor(string $color = "", int $offset = 40, bool $in
         return "#" . $fg_color;
     }
 
+
+    public static function isColorDark(string $color = ""): bool
```
Comment (cedric-anne) [inline src/Toolbox.php:3010]: This parameter is mandatory, it should not have a default value.

```suggestion
    public static function isColorDark(string $color): bool
```

File: src/Toolbox.php [hunk 2]
```diff
@@ -3006,6 +3006,35 @@ public static function getFgColor(string $color = "", int $offset = 40, bool $in
...
+            $color = str_replace("#", "", $color);
+
+            // if transparency present, get only the color part
+            if (strlen($color) === 8 && preg_match('/^[a-fA-F0-9]+$/', $color)) {
+                $tmp = $color;
+                $alpha = hexdec(substr($tmp, 6, 2));
+                $color = substr($color, 0, 6);
+
+                if ($alpha <= 100) {
+                    return "inherit";
```
Comment (cedric-anne) [inline src/Toolbox.php:3022]: Method is supposed to return a boolean, not a string. It will fail with a PHP error.

File: src/Toolbox.php [hunk 3]
```diff
@@ -3006,6 +3006,35 @@ public static function getFgColor(string $color = "", int $offset = 40, bool $in
...
+            }
+
+            $color_inst = new Color($color);
+            return $color_inst->isLight();
+        }
+
+        return false;
+    }
+
+    public static function isColorLight(string $color = ""): bool
```
Comment (cedric-anne) [inline src/Toolbox.php:3033]:
```suggestion
    public static function isColorLight(string $color): bool
```

**Comment by AdrienClairembault** [comment]
image

It feel weird to mix bordered and non bordered items.
Is is done on purpose to only add the new border to the big number items ? 
Shouldn't it be on every card ?

**Comment by orthagh** [comment]
Tried on other cards, was not convinced.
I may work more on this, but pies are always bad regarding the styles

**Review by AdrienClairembault — APPROVED** [review]

**Review by cedric-anne — APPROVED** [review]

**Pull request merged**